pages/events/[slug]/index.js

```diff
@@ -79,6 +91,17 @@ const Event = ({ event, error }) => {
     }
   }
 
+  const handleCreatorSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      await platform.event.patch(event._id, {
+        createdBy : (event.createdBy || '').concat(creatorToAdd)
```

Comment:
This is not supported. You can set an event.managedBy which is an array of users though.